x-pack/test/ui_capabilities/security_and_spaces/tests/foo.ts

```diff
@@ -79,8 +66,14 @@ export default function fooTests({ getService }: FtrProviderContext) {
           case 'everything_space_read at nothing_space':
           case 'nothing_space_all at everything_space':
           case 'nothing_space_read at everything_space':
-            expect(uiCapabilities.success).to.be(false);
-            expect(uiCapabilities.failureReason).to.be(GetUICapabilitiesFailureReason.NotFound);
+            expect(uiCapabilities.success).to.be(true);
+            expect(uiCapabilities.value).to.have.property('foo');
+            expect(uiCapabilities.value!.foo).to.eql({
+              create: false,
+              edit: false,
+              delete: false,
+              show: false,
+            });
```

Comment:
As @kobelb explained to me:

> so, we used to be getting a 404 because we used /app/kibana to load the ui capabilities: https://github.com/elastic/kibana/pull/51438/files#diff-32f6b7c96a3f2ac5ee5c7c132dfeeff2L60
This hit the following http request middleware which only applies to routes that start with /app https://github.com/elastic/kibana/blob/ad356f5c1186140cb6128a8cba3f1e04752d3dd8/x-pack/plugins/security/server/authorization/app_authorization.ts#L48
Since we're now using /api/core/capabiliites, and the users in that test suite are "authenticated" but not "authorized" we get back a 200 from the /api/core/capabiliites call with all of the ui capabilities disabled